library/LuaApi.cpp

```diff
@@ -3577,6 +3577,67 @@ static int internal_md5file(lua_State *L)
     }
 }
 
+/**************************
+ * Wrappers for Alias API *
+ **************************/
+
+static int internal_getAliasCommand(lua_State *L)
+{
+    const char *name = luaL_checkstring(L, 1);
+    std::string ret = Core::getInstance().GetAliasCommand(name);
+    if (!ret.empty())
+        lua_pushstring(L, ret.c_str());
+    else
+        lua_pushnil(L);
+    return 1;
+}
+
+static int internal_listAliases(lua_State *L)
+{
+    auto aliases = Core::getInstance().ListAliases();
+
+    lua_newtable(L);
+    for (const auto& p : aliases)
+    {
+        Lua::Push(L, p.first.c_str()); // alias
+        Lua::PushVector(L, p.second);
+        lua_settable(L, -3);
+    }
+    return 1;
+}
+
+static int internal_isAlias(lua_State *L)
+{
+    const char *name = luaL_checkstring(L, 1);
+    lua_pushboolean(L, Core::getInstance().IsAlias(name));
+    return 1;
+}
+
+static int internal_addAlias(lua_State *L)
+{
+    std::vector<std::string> alias;
+    Lua::GetVector(L, alias, 1);
+
+    if (!alias.empty())
+    {
+        std::string name = alias[0];
```

Comment:
I think we should match the AddAlias() signature instead by taking two arguments: `name` (a string) and `arguments` (a list of arguments). It looks like you're retrieving the alias name from the first element of the list here, which doesn't match the C++ API.

Reply:
I'll adjust.